schema/render.go

```diff
@@ -165,7 +162,7 @@ func Render(document *parser.Document) (string, error) {
 			}
 
 			newSchema.SchemaProps.Properties = properties
-			if len(level.Children) > 0 {
+			if len(level.Children) > 0 && !(paths.Path{}).WithProperty("global").IsSubPathOf(level.Path) {
```

Comment:
**Suggestion:** Consider adding a comment to clarify the purpose of `&& !(paths...`. Its function may not be immediately clear at first glance.